cockatrice/src/window_main.cpp

```diff
@@ -60,7 +60,7 @@
 #define GITHUB_PAGES_URL "https://cockatrice.github.io"
 #define GITHUB_CONTRIBUTORS_URL "https://github.com/Cockatrice/Cockatrice/graphs/contributors?type=c"
 #define GITHUB_CONTRIBUTE_URL "https://github.com/Cockatrice/Cockatrice#cockatrice"
-#define GITHUB_TRANSLATOR_RECOGNIZE_URL "https://github.com/Cockatrice/Cockatrice/wiki/Translators"
+#define GITHUB_TRANSIFEX_TRANSLATORS_URL "https://github.com/Cockatrice/Cockatrice/wiki/Translator-Hall-of-Fame"
```

Comment:
The link in the old version of the code is broken. I guess no one ever
clicked it, or if they did, they didn't report anything.

On Sat, Jul 9, 2016 at 11:58 AM tooomm <notifications@github.com> wrote:

> In cockatrice/src/window_main.cpp
> <https://github.com/Cockatrice/Cockatrice/pull/1987#discussion_r70168748>:
>
> > @@ -60,7 +60,7 @@
> >  #define GITHUB_PAGES_URL "https://cockatrice.github.io"
> >  #define GITHUB_CONTRIBUTORS_URL "https://github.com/Cockatrice/Cockatrice/graphs/contributors?type=c"
> >  #define GITHUB_CONTRIBUTE_URL "https://github.com/Cockatrice/Cockatrice#cockatrice"
> > -#define GITHUB_TRANSLATOR_RECOGNIZE_URL "https://github.com/Cockatrice/Cockatrice/wiki/Translators
>
> "
> > +#define GITHUB_TRANSIFEX_TRANSLATORS_URL "https://github.com/Cockatrice/Cockatrice/wiki/Translator-Hall-of-Fame"
>
> That's the original working link. We just have the Translators Hall of
> Fame
> <https://github.com/Cockatrice/Cockatrice/wiki/Translator-Hall-of-Fame>
> to name them. It has always been like that. Zach created it like one year
> ago.
>
> I'm not sure if I like that page name though...
>
> Maybe if was broken all the time? Or Can't tell for sure.
>
> —
> You are receiving this because you commented.
>
> Reply to this email directly, view it on GitHub
> <https://github.com/Cockatrice/Cockatrice/pull/1987/files/6d9bc2347ae0920adc664fedf210f6d56b02aa8a#r70168748>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAA5NHQvR8iUfq35Hxk99tgoM3LDizc0ks5qT8UxgaJpZM4Ij9db>
> .
>